docs/examples/hierarchies/python/app.py

```diff
@@ -0,0 +1,25 @@
+from oso import Oso
+from polar import Relation
+
+oso = Oso()
+
+# Register the Organization class
+oso.register_class(
+    Organization,
+    types={
+        "id": str,
+    },
+    build_query=build_query_cls(Organization),
```

Comment:
where is `build_query_cls()` defined?

Reply:
ignore this — left the comment before I realized that the examples aren't intended to be runnable